fluid/understand_sentiment_by_dynrnn.py

```diff
@@ -0,0 +1,193 @@
+import argparse
```

Comment:
We have a old one understand_sentiment_dynamic_rnn.py, I think we can delete that one and replace with this one?